code/modules/mob/living/simple_animal/friendly/drone/drone_tools.dm

```diff
@@ -0,0 +1,84 @@
+
+/obj/item/storage/drone_tools
+	name = "built-in tools"
+	desc = "Access your built in tools"
+	icon = 'icons/hud/screen_drone.dmi'
+	icon_state = "tool_storage"
+	item_flags = ABSTRACT
+	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF
+
+/obj/item/storage/drone_tools/Initialize()
+	. = ..()
+	ADD_TRAIT(src, TRAIT_NODROP, ABSTRACT_ITEM_TRAIT)
+
+/obj/item/storage/drone_tools/ComponentInitialize()
+	. = ..()
+	var/static/list/drone_builtins = list(	/obj/item/crowbar/drone, /obj/item/screwdriver/drone, /obj/item/wrench/drone, \
+											/obj/item/weldingtool/drone, /obj/item/wirecutters/drone)
+	var/datum/component/storage/STR = GetComponent(/datum/component/storage)
+	STR.max_combined_w_class = 40
+	STR.max_w_class = WEIGHT_CLASS_NORMAL
+	STR.max_items = 5
+	STR.rustle_sound = FALSE
+	STR.set_holdable(drone_builtins)
+
+
+/obj/item/storage/drone_tools/PopulateContents()
+	var/list/builtintools = list()
+	builtintools += new /obj/item/crowbar/drone(src)
+	builtintools += new /obj/item/screwdriver/drone(src)
+	builtintools += new /obj/item/wrench/drone(src)
+	builtintools += new /obj/item/weldingtool/drone(src)
+	builtintools += new /obj/item/wirecutters/drone(src)
+
+	for(var/_tool in builtintools)
+		var/obj/item/tool = _tool
+		tool.AddComponent(/datum/component/holderloving, src, TRUE)
+
+
+///// DRONE TOOLS //////
+
+/obj/item/crowbar/drone
+	name = "built-in crowbar"
+	desc = "A crowbar built into your chassis."
+	icon = 'icons/obj/items_cyborg.dmi'
+	icon_state = "crowbar_cyborg"
+	inhand_icon_state = "crowbar"
+
+/obj/item/screwdriver/drone
+	name = "built-in screwdriver"
+	desc = "A screwdriver built into your chassis."
+	icon = 'icons/obj/items_cyborg.dmi'
+	icon_state = "screwdriver_cyborg"
+	inhand_icon_state = "screwdriver"
+	random_color = FALSE
+
+
+/obj/item/screwdriver/drone/worn_overlays(isinhands = FALSE, icon_file)
+	. = list()
+	if(isinhands)
+		var/mutable_appearance/M = mutable_appearance(icon_file, "screwdriver_head")
```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names

Reply:
I think this is a bit too far but ok